api/src/main/java/org/openmrs/module/quarterlyreporting/regimenhistory/RegimenUtils.java

```diff
@@ -15,13 +15,15 @@
 import org.openmrs.api.OrderService;
 import org.openmrs.api.context.Context;
 import org.openmrs.util.OpenmrsUtil;
+//import org.openmrs.module.mohorderentrybridge.api.MoHOrderEntryBridgeService;
 
 public class RegimenUtils {
 
     protected final Log log = LogFactory.getLog(getClass());
     
     public static RegimenHistory getRegimenHistory(Patient patient) {
-        return new RegimenHistory(Context.getOrderService().getDrugOrdersByPatient(patient));
+     // return new RegimenHistory(Context.getService(MoHOrderEntryBridgeService.class).getDrugOrdersByPatient(patient));

```

Comment:
will this method work any longer than it was intended to work after you have hard coded it to always return null, have you considered looking at for-example what i have done for https://github.com/rwanda-rbc-emr/openmrs-module-drugorderexport/blob/master/api/src/main/java/org/openmrs/module/regimenhistory/RegimenUtils.java#L24 ?